### PR TITLE
Adding sample slides, for different Python projects

### PR DESCRIPTION
**Pull request by ranms25**
Adding sample slides, for different Python projects. I tried to change some things and try to make different variations, so that in every project there are changes in terms of writing, but the idea is the same.

### CODE REVIEW

File: docker/opensource.md [hunk 1]
```diff
@@ -87,5 +87,102 @@ $ docker run -it --rm --workdir /opt -v$(pwd):/opt ubuntu:22.10 bash
...
+## Cosmo-Tech
+{id: docker-python-Cosmo-Tech}
+
+
+* [Cosmo-Tech](https://github.com/Cosmo-Tech/CosmoTech-Acceleration-Library)
+
+```
+$ git clone https://github.com/Cosmo-Tech/CosmoTech-Acceleration-Library.git
+$ cd CosmoTech-Acceleration-Library
+$ docker run -t -i --name cosmotech-acceleration-library-dev -v /opt/CosmoTech-Acceleration-Library -v /C:/Users/YourUserName/CosmoTech-Acceleration-Library:/opt python:3.11 bash
```
Comment (szabgab) [inline docker/opensource.md:99]: Why are there two -v parameters? The 2nd one is for Windows, right?

File: docker/opensource.md [hunk 2]
```diff
@@ -87,5 +87,102 @@ $ docker run -it --rm --workdir /opt -v$(pwd):/opt ubuntu:22.10 bash
...
+
+# cd opt
```
Comment (szabgab) [inline docker/opensource.md:101]: if -w /opt is provided on the command line of docker run then there is no need for this.

File: docker/opensource.md [hunk 3]
```diff
@@ -87,5 +87,102 @@ $ docker run -it --rm --workdir /opt -v$(pwd):/opt ubuntu:22.10 bash
...
+
+## mobility
+{id: docker-python-mobility}
+
+
+* [mobility](https://github.com/mobility-team/mobility)
+```
+$ git clone https://github.com/mobility-team/mobility.git
+$ cd mobility
+$ docker run -t -i --name mobility-dev -v /opt/mobility -v /C:/Users/YourUserName/mobility:/opt python:3.9 bash
```
Comment (szabgab) [inline docker/opensource.md:119]: as above

File: docker/opensource.md (hunk 1 above)
Comment (ranms25) [reply docker/opensource.md:99]: Maybe is realted to that I run it on Git Bash?

I tried with w but get erroes, and try to dix it but it didnt work:
docker run -it --name cosmotech-acceleration-library-dev -w /opt -v /C:/Users/Ran/CosmoTech-Acceleration-Library:/opt python:3.11 bash               
docker: Error response from daemon: the working directory 'C:/Program Files/Git/opt' is invalid, it needs to be an absolute path.
See 'docker run --help'.

docker run -it --name cosmotech-acceleration-library-dev -w /opt -v %cd%:/opt python:3.11 bash
docker: Error response from daemon: the working directory 'C:/Program Files/Git/opt' is invalid, it needs to be an absolute path.
See 'docker run --help'.

File: docker/opensource.md (hunk 2 above)
Comment (ranms25) [reply docker/opensource.md:101]: again it didn't worked for me

File: docker/opensource.md (hunk 3 above)
Comment (ranms25) [reply docker/opensource.md:119]: the demand was for python_requires=">=3.9"
So I chose 3.9, becasue somtimes the update version did errors in the tests

File: docker/opensource.md (hunk 1 above)
Comment (ranms25) [reply docker/opensource.md:99]: And I'm quite confused, what is meant to do more for Linux? I was working with Windows, but I ran everything through Git Bash and it worked fine for me with no problems.

Comment (szabgab) [reply docker/opensource.md:99]: I could not run docker from git bash properly either. Only CMD and PowerShell

**Pull request merged**